cypress/support/commands.js

```diff
@@ -30,3 +30,25 @@ Cypress.Commands.add("loadMoreVideos", () => {
 Cypress.Commands.add("closeSearchModal", () => {
   cy.get(".dimmer").click({ force: true });
 });
+
+Cypress.Commands.add("clickWatchedVideoListTab", () => {
+  cy.get("#watched-video-button").click();
+});
+
+Cypress.Commands.add("clickWatchLaterVideoListTab", () => {
+  cy.get("#watch-later-video-button").click();
+});
+
+Cypress.Commands.add("clickVideoWatchButton", () => {
+  cy.get(".video-item__watched-button").eq(0).click();
+});
+
+Cypress.Commands.add("clickVideoDeleteButton", (confirmButtonClick) => {
+  cy.get(".video-item__delete-button").eq(0).click();
+
+  cy.on("window:confirm", (text) => {
+    console.log(text);
+    expect(text).to.contains("정말로 삭제하시겠습니까?");
+    return confirmButtonClick;
```

Comment:
return은 안해주셔도 괜찮아보여요!
```suggestion
    expect(text).to.contains("정말로 삭제하시겠습니까?");
```

Reply:
return 문에 boolean 값을 넣어주는걸로 comfirm창의 `확인` 버튼과 `취소` 버튼을
누를 수 있다고 해서 인자로 받아 넣어주었었는데 혹시 제가 잘못 알고 있었던걸까요?
리턴을 지워주니 오류가 나기에 다시 여쭈어봅니다 😭

Reply:
> return 문에 boolean 값을 넣어주는걸로 comfirm창의 `확인` 버튼과 `취소` 버튼을 누를 수 있다고 해서 인자로 받아 넣어주었었는데 혹시 제가 잘못 알고 있었던걸까요? 리턴을 지워주니 오류가 나기에 다시 여쭈어봅니다 😭

cypress window:confirm에 return이 필요한걸 깜빡했네요 🙇  
말씀하신대로 return을 넣어줘야 confirm창을 확인이나 취소할 수 있네요. 죄송합니다!